src/jit/compiler.cpp

```diff
@@ -3990,7 +3990,8 @@ void Compiler::compInitDebuggingInfo()
 
 void Compiler::compSetOptimizationLevel()
 {
-    bool     theMinOptsValue;
+    bool theMinOptsValue;
```

Comment:
Is any reason why we can't move it lower when it is used?
line 4011:
```
#ifdef DEBUG
unsigned jitMinOpts = JitConfig.JitMinOpts();
```

Reply:
I've did this initially and while MSVC is happy with it, the Linux builds (which are probably using clang) are not.
```
18:35:02 /mnt/j/workspace/dotnet_coreclr/master/linux-musl-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/jit/compiler.cpp:3998:9: error: cannot jump from this goto statement to its label
18:35:02         goto _SetMinOpts;
18:35:02         ^
18:35:02 /mnt/j/workspace/dotnet_coreclr/master/linux-musl-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/src/jit/compiler.cpp:4010:14: note: jump bypasses variable initialization
18:35:02     unsigned jitMinOpts = JitConfig.JitMinOpts();
18:35:02              ^
18:35:02 1 error generated.
```